tests/factories/application_instance.py

```diff
@@ -4,7 +4,7 @@
 from lms import models
 from tests.factories.attributes import OAUTH_CONSUMER_KEY, SHARED_SECRET
 
-ApplicationInstance = make_factory(  # pylint:disable=invalid-name
```

Comment:
:man_shrugging: This was a false-positive from PyLint, apparently it now no longer complains about it in this case. Great